Add docs for `{NavigateEvent,PopStateEvent}.hasUAVisualTransition` (#36705)
mdn/content: files/en-us/web/api/navigateevent/index.md

  - : Returns the {{domxref("FormData")}} object representing the submitted data in the case of a `POST` form submission, or `null` otherwise.
- {{domxref("NavigateEvent.hashChange", "hashChange")}} {{ReadOnlyInline}} {{Experimental_Inline}}
  - : Returns `true` if the navigation is a fragment navigation (i.e. to a fragment identifier in the same document), or `false` otherwise.

> - {{domxref("NavigateEvent.hasUAVisualTransition", "hashChange")}} {{ReadOnlyInline}} {{Experimental_Inline}}

Author: ```suggestion
- {{domxref("NavigateEvent.hasUAVisualTransition", "hasUAVisualTransition")}} {{ReadOnlyInline}} {{Experimental_Inline}}
```

>   - : Returns `true` if the navigation has been performed a visual transition before dispatching this event by the user agent, or `false` otherwise.

Reviewer: ```suggestion
  - : Returns `true` if the user agent performed a visual transition for this navigation before dispatching this event, or `false` otherwise.
```
